autoload/gh/github/repos.vim

```diff
@@ -4,6 +4,19 @@
 
 let s:Base64 = vital#gh#import('Data.Base64')
 
+let s:assignee_list_query =<< trim END
+{
+  repository(name: "%s", owner: "%s") {
+    assignableUsers(first: 10) {
```

Comment:
We need to get all of repository's users
GraphQL can get 100 members but it's  maybe not all in repository, so we need to use cursor to get when repository has over 100 members

Reply:
@skanehira 
sorry, i don't understand why we need to get users who are not assignable for the issue.
i guess getting only `assignableUsers` like this way is enough to show the list of assignable users on quickpick buffer.
something of my understanding is wrong...?

Reply:
Oh, sorry my explanation is not good.
I mean I think we need to get assignable users  for the issue more than 10.
But in GitHub specification, assign for 1 issue is until 10 users.

Reply:
sorry for replying late... lol

ok, i got understand what you mean.
tried to fix by [this commit](https://github.com/skanehira/gh.vim/pull/119/commits/3d450573c3d347892641c7ae2618046efe674b72).

Please check it again :)